src/wallet/feebumper.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_WALLET_FEEBUMPER_H
+#define BITCOIN_WALLET_FEEBUMPER_H
+
+#include <primitives/transaction.h>
+
+class CWallet;
+class uint256;
+
+class CFeeBumper
+{
+public:
+    enum class BumpFeeResult
```

Comment:
I think the less verbose `BumpFeeResult::OK` would be preferable to `CFeeBumper::BumpFeeResult::OK`

Could move enum above class to accomplish this. Or could change "enum class" to "enum" for `CFeeBumper::OK`.